docs/user/installation.rst

```diff
@@ -68,6 +69,17 @@ Optional dependencies:
    zypper in bash-completion jq
    zypper in python3-psycopg2 python3-pymongo python3-sleekxmpp
 
+Docker (beta)
+^^^^^^^^^^^^^
+
+**ATTENTION** Currently docker version can differ from intelmq version namings.
```

Comment:
I'm not sure I get the meaning of that sentence. Docker version is for example 1.5 and IntelMQs version is 2.2 - do you mean the version of the docker container does not follow the same version schema as IntelMQ does? Then I would write something like `Currently the version of the docker container published on docker hub is not tagged with the same version as the IntelMQ release it contains. It is tagged with version foo, but contains IntelMQ version bar.`

Reply:
This is only the case during the beta phase. After the beta is intended to follow the original intelmq versioning.

You can find a good reference here https://hub.docker.com/_/alpine or https://hub.docker.com/_/mariadb